### ISSUE TITLE
Freedom of choosing optimizer and learning rate in the model 

### ISSUE THREAD

**Issue by stacezwu** [keras-team/autokeras#1560]
### Feature Description
Build a model with a specific optimizer and learning rate instead of searching through the space.

### Code Example

```python
auto_model = ak.AutoModel(
      inputs=input_node, outputs=output_node, optimizer="adam", learning_rate=0.001, objective="accuracy", 
      tuner="greedy", overwrite=True, max_trials=20)
```

### Reason

It would be useful when a certain optimizer is already decided on from previous search and the trials can focus more on other hyperparameters for time efficiency.
